src/Microsoft.Content.Build.MerfToSDPForJava.SDP.DataContracts/ItemType.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace Microsoft.Content.Build.MerfToSDPForJava.SDP.DataContracts
+{
+    using System;
+    public enum ItemType
+    {
+        Default,
+        Toc,
+        Assembly,
+        Namespace,
+        Class,
+        Interface,
+        Struct,
+        Delegate,
+        Enum,
+        Field,
+        Property,
+        Event,
+        Constructor,
+        Method,
+        Operator,
+        Container,
+        AttachedEvent,
+        AttachedProperty
```

Comment:
I think we do not have "Assembly", "Struct", "Operator", "AttachedEvent", "AttachedProperty"

Reply:
sure

Reply:
Updated